### PR TITLE
Docker compose consensus layer config

### PR DESCRIPTION
**Pull request by bacv**
Correctly configure consensus layer in bootstrap node and other nomos nodes. These are a follow up changes to https://github.com/logos-co/nomos-node/pull/467#discussion_r1365410626 comment.

Consensus layer configuration now has all nodes (the number of nodes is controlled by `DOCKER_COMPOSE_LIBP2P_REPLICAS` in `.env`) created by compose file.

### CODE REVIEW

File: compose.yml [hunk 1]
```diff
@@ -31,10 +34,11 @@ services:
       - LIBP2P_REPLICAS=${DOCKER_COMPOSE_LIBP2P_REPLICAS:-1}
       - ETCDCTL_ENDPOINTS=${DOCKER_COMPOSE_ETCDCTL_ENDPOINTS:-etcd:2379}
       - ETCDCTL_API=${DOCKER_COMPOSE_ETCDCTL_API:-3}
-      - NODE_MASK=${DOCKER_COMPOSE_LIBP2P_NODE_KEY_MASK:-2000000000000000000000000000000000000000000000000000000000000000}
+      - BOOTSTRAP_NODE_KEY=${DOCKER_COMPOSE_BOOSTRAP_NET_NODE_KEY:-1000000000000000000000000000000000000000000000000000000000000000}
```
Comment (youngjoon-lee) [inline compose.yml:37]: It seems that this `BOOTSTRAP_NODE_KEY` is not used by the `run_nomos_node.sh`and any sub-scripts.

**Review by youngjoon-lee — APPROVED** [review]

File: compose.yml (hunk 1 above)
Comment (danielSanchezQ) [reply compose.yml:37]: Im curious, why is it negative?

**Review by danielSanchezQ — APPROVED** [review]

File: compose.yml (hunk 1 above)
Comment (bacv) [reply compose.yml:37]: This is a notation for a default value if the one from `.env` is absents. Agree that it's quite misleading on first glance.

Comment (bacv) [reply compose.yml]: > It seems that this BOOTSTRAP_NODE_KEY is not used by the run_nomos_node.shand any sub-scripts.

Thanks for spotting that, fixed.

**Pull request merged**